GLAA.Web/Controllers/DefaultController.cs

```diff
@@ -64,17 +72,57 @@ protected virtual string GetLastViewPath(FormSection section)
             return View(viewPath, viewModel);
         }
 
+        /// <summary>
+        /// Return a view for the requested action name. If this view can't be viewed, it will seek backwards 
+        /// through the pages until one that can be viewed is found. If no view can be found then the last view 
```

Comment:
Last view as in the one which was just viewed? or the last in the section?

Reply:
I think I've been universal in my use of "last" as being "last in the section". The intention is that this usage mirrors Linq's `.Last()`.